### DISCUSSION TITLE
Dual Stack (IPV4-IPV6) support on vmware UPI OKD4.7 fails

### DISCUSSION THREAD

**Discussion by vglaiju** [okd-project/okd#828] [Q&A]
**Describe the bug**
Brought up an OKD4.7 with default ovs network, The masters & workers and operators come up fine.
Then for enabling IPV4-IPV6 dual stack support we apply the ovn patch as below.
After applying ovn-kubernetes patch, per https://docs.okd.io/latest/networking/ovn_kubernetes_network_provider/converting-to-dual-stack.html
the pods are crashing with sandbox error and unable to receive IP address assignment and reporting dependent issue on multus pod when we describe the pod.
A bridge br-ex interface was enabled to all workers for global IPV6 support on all worker nodes.

**Version**
Version : OKD4.7
vmware baremetal : UPI
**How reproducible**
Reproducible after applying ovn-kuberenetes patch. 

**Log bundle**
following is the ovn-kubenode log.

I0812 10:46:31.528246   30941 gateway_init.go:162] Initializing Gateway Functionality
I0812 10:46:31.528492   30941 gateway_localnet.go:183] Node local addresses initialized to: map[10.130.0.2:{10.130.0.0 fffffe00} 127.0.0.1:{127.0.0.0 ff000000} 167.254.227.187:{167.254.227.0 ffffff00} 2001:0:0:3::2:{2001:0:0:3:: ffffffffffffffff0000000000000000} ::1:{::1 ffffffffffffffffffffffffffffffff} fe80::9854:c9ff:fe40:4800:{fe80:: ffffffffffffffff0000000000000000} fe80::b462:e9ff:fe8f:4544:{fe80:: ffffffffffffffff0000000000000000}]
I0812 10:46:31.528629   30941 helper_linux.go:74] Found default gateway interface br-ex 167.254.227.254
F0812 10:46:31.528697   30941 ovnkube.go:130] failed to get default gateway interface

We also observed that when control-planes reboots, "openshift-etcd' namespace is able to bring up all the expected 3 containers ( etcd-metrics, etcd, etcdctl) on each control -plane nodes however the other namespace "openshift-etcd-operator" is not able to bring up "etcd-operator" container due to sandbox was not able to get created in control-planes and pods stuck in ContainerCreating state.

**Comment by vrutkovs** [comment]
Please attach [log bundle](https://docs.okd.io/latest/installing/installing-troubleshooting.html#installation-bootstrap-gather_installing-troubleshooting)

>Brought up an OKD4.7 with default ovs network

You mean OpenshiftSDN? OKD's default is OVN
